resources/css/components/cards.css

```diff
@@ -10,6 +10,39 @@
     .badge {
       @apply absolute left-4 top-4 z-2;
     }
+    &-carousel{
+      @apply relative h-full;
+
+      carousel-slide{
+        @apply inline-block h-full opacity-0 transition-all;
+        &.active{
+          @apply opacity-100;
+        }
+        img{
+          @apply absolute top-0;
+        }
+      }
+      &-navigation{
+        @apply absolute top-1/2 w-full;
```

Comment:
режим `overlay`

```suggestion
      &-navigation{
        @apply absolute top-1/2 w-full z-10;
```

![Снимок экрана 2024-06-04 в 11 38 26](https://github.com/moonshine-software/moonshine/assets/12373059/7d96d9ad-f227-4b29-8954-352b7f914177)

навигация слайдов под тестовым блоком

Reply:
![image](https://github.com/moonshine-software/moonshine/assets/886244/516c5dec-d461-4a50-94cc-dededb38daae)
добавил